httpserver-rs/src/settings.rs

```diff
@@ -252,6 +264,16 @@ pub fn load_settings(values: &HashMap<String, String>) -> Result<ServiceSettings
         ));
     }
 
+    // accept cache-control header values
+    if let Some(cache_control) = values.get("cache_control") {
```

Comment:
Let's update the settings documentation as well: https://github.com/wasmCloud/capability-providers/blob/main/httpserver-rs/settings.md